src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -1189,6 +1195,13 @@ public SocketAsyncContext(SafeSocketHandle socket)
             _sendQueue.Init();
         }
 
+        public bool PreferInlineCompletions
+        {
+            [PreserveDependency("get_PreferInlineCompletions", "System.Net.Sockets.Socket")]
+            [PreserveDependency("set_PreferInlineCompletions", "System.Net.Sockets.Socket")]
```

Comment:
The get_ one shouldn't be necessary, since it's explicitly used here.
The set_ one could use a comment indicating this is temporary until we either a) make the property public or b) delete it.